systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -1,5 +1,5 @@
 classdef QPLocomotionPlan < QPControllerPlanMatlabImplementation
-  properties(GetAccess=private, SetAccess=private)
+  properties(GetAccess=protected, SetAccess=protected)
```

Comment:
That's fine. I was mostly just establishing for myself the ways in which these properties were being changed.